manager/src/main/java/io/aklivity/zilla/manager/internal/commands/install/ZpmRepository.java

```diff
@@ -25,6 +27,7 @@
 public final class ZpmRepository
 {
     public String location;
+    public String host;
```

Comment:
Assuming this is temporary until we are using standard maven credentials approach instead of alternative added for ivy credentials.